client/src/pages/Records.jsx

```diff
@@ -0,0 +1,83 @@
+import Accordion from 'react-bootstrap/Accordion';
+import Button from 'react-bootstrap/esm/Button';
+import { useState, useEffect, useCallback } from 'react'
+import AccordionItems from '../components/AccordionItems.jsx';
+
+export default function RecordsView() {
+  const [records, setRecords] = useState([]);
+  const [page, setPage] = useState(0);
+  const [loading, setLoading] = useState();
+  const [endOfRecords, setEndOfRecords] = useState(false);
+
+  /**
+   * Callback function to request the current page from the api.
+   * In a useCallback to prevent infinite loop when loading.
+   * Returns an organized array of record objects after the response is sorted.
+   * If their are no more pages sets endOfRecords to true and ends loading.
+   */
+  const getRecords = useCallback(async () => {
+    setLoading(true)
+    try {
+      const res = await fetch(`/api/records/${page}`);
+      if (!res.ok) throw new Error(`Could not load results ${res.status}`);
+      const myrecords = await res.json();
+      if (!myrecords.nextPage) {
+        setEndOfRecords(true);
+        setLoading(false);
+        return
+      };
+      setPage(myrecords.nextPage);
+      const sortedRecords = sortRecords(myrecords);
+      setLoading(false);
+      return sortedRecords;
+    } catch (err) {
```

Comment:
It's not a good idea to just ignore this error. At least log it, and eventually display a message to the user.